### PR TITLE
working in 2 tickets in this single branch

### PR DESCRIPTION
**Pull request by louidorjp**
Ticket1: https://pihemr.atlassian.net/jira/software/c/projects/HAI/boards/156?selectedIssue=HAI-886
Ticket2: https://pihemr.atlassian.net/jira/software/c/projects/HAI/boards/156?selectedIssue=HAI-865

if_other_service
Screenshot from 2023-10-20 16-27-46

Please review this PR

### CODE REVIEW

File: configuration/messageproperties/messages_en.properties [hunk 1]
```diff
@@ -3519,3 +3519,10 @@ pih.app.hiv.viralLoadHistory.lld=Lower limit of detection (copies/ml)
 
 emr.dataClerk.required=Data Clerk (required)
 emr.psychologue.required=Psychologue (required)
+
+pihcore.mentalhealth.if.other.service=If other service?
+pih.app.mental.health.emergencyRoom=Emergency room
+pih.app.mental.health.surgery=Surgery
+pih.app.mental.health.internal=Internal Medicine
+pih.app.mental.health.other=Other
```
Comment (lnball) [inline configuration/messageproperties/messages_en.properties:3527]: You don't need new messages when they exist.  Please re-use these:

- pihcore.other
- pihcore.location.ed
- pihcore.location.surgery
- pihcore.location.intMed

**Review by lnball — CHANGES_REQUESTED** [review]
Please update with existing message properties.  Thanks.  Otherwise it looks good.

**Comment by louidorjp** [comment]
@lnball existing message properties added

**Comment by lnball** [comment]
Looks good @louidorjp .  Thanks.

**Pull request merged**